### DISCUSSION TITLE
Accidentally skipped start wizard, can't create profile

### DISCUSSION THREAD

**Discussion by JollyGrin** [beakerbrowser/beaker#1754] [General]
Greetings!

I accidentally closed out of the start wizard before creating an account. Upload starting the browser again, it launched except I have no profile

![image](https://user-images.githubusercontent.com/28109194/85795895-4f51c800-b739-11ea-9632-83c7bc17f7a5.png)

How do I create a profile?

**Comment by pfrazee** [comment]
We fixed this by resetting their data directory (support via twitter!). Definitely shouldnt have happened so that's an odd bug

**Reply by evelyn-fyi** [reply]
Hello! Still having this issue, unfortunately.

**Comment by tkbremnes** [comment]
This happened to me as well. The wizard told me that the profile-creation was optional, so I closed it thinking I could do that later.

**Reply by pfrazee** [reply]
I was able to repro recently as well. Looks a bug, so I've opened an issue to tackle in the next release.

Thanks for sharing!

**Comment by leeclarke** [comment]
Just happened to me as well. Strange part is I entered a user name and thought I hit create button... I skipped the image and bio  planning to come back to it later but no profile.   Not sure if it 's helpful but, just in case!

**Reply by leeclarke** [reply]
Oh this might be handy too:
![image](https://user-images.githubusercontent.com/288559/90317910-99198c00-defa-11ea-8332-6099880e9060.png)
